File: NewsDaily2/app/src/main/java/com/andruid/magic/newsdaily/model/AudioNews.java
package com.andruid.magic.newsdaily.model;

import com.andruid.magic.newsloader.model.News;

public class AudioNews {
    private final String uri;
    private final News news;

    public AudioNews(String uri, News news) {
        this.uri = uri;
        this.news = news;
    }

    public String getUri() {
        return uri;
    }

    public News getNews() {
        return news;
    }
}

File: NewsDaily/app/src/main/java/com/death14stroke/newsdaily/data/model/AudioNews.kt
package com.death14stroke.newsdaily.data.model

import com.death14stroke.newsloader.data.model.News

/**
 * Data model for news item in the audio playlist
 * @property uri uri of the text to speech output
 * @property news metadata of the news object
 * @see News
 */
data class AudioNews(
    val uri: String,
    val news: News
)